Displays an error message when the $SERIES file does not exist and when run
in verbose mode.

Closes: #369908

Status wrt upstream: not submitted.


---
 quilt/scripts/patchfns.in |    3 +++
 1 file changed, 3 insertions(+)

--- a/quilt/scripts/patchfns.in
+++ b/quilt/scripts/patchfns.in
@@ -323,6 +323,9 @@ cat_series()
 		sed -e '/^#/d' -e 's/^[ '$'\t'']*//' \
 		    -e 's/[ '$'\t''].*//' -e '/^$/d' $SERIES
 	else
+		if [ "$opt_verbose" ]; then
+			echo "series file not found: $SERIES" >&2
+		fi
 		return 1
 	fi
 }
